### PR TITLE
Add problems in greedy algorithms

### PR DESCRIPTION
**Pull request by donaldong**
Here are some problems can be solved using greedy algorithms.

I did not keep tracking those things until I got to around 2000 pts on Kattis. So there are many good easy to medium out there on Kattis which I did not include.

### CODE REVIEW

File: greedy.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Greedy Algorithms
+=================
+Here are some problems can be solved using a greedy strategy. A greedy approach
```
Comment (FernandoCaudillo10) [inline greedy.md:3]: I would suggest adding a clear definition of what Greedy algorithms entail. Maybe we can consider linking to a website that would further explain the topic?

Comment (FernandoCaudillo10) [reply greedy.md:3]: Or should we leave explaining the paradigm during progteam meetings?

Comment (donaldong) [reply greedy.md:3]: Yeah, I agree that It could use a little bit more explanation here.

I updated the intro, what do you think? @FernandoCaudillo10

**Review by FernandoCaudillo10 — APPROVED** [review]

File: greedy.md (hunk 1 above)
Comment (FernandoCaudillo10) [reply greedy.md:3]: @donaldong I think this is great!

**Pull request merged**